metadata/org.testcontainers/testcontainers/1.19.8/resource-config.json

```diff
@@ -0,0 +1,27 @@
+{
+    "resources": {
+        "includes": [
+            {
+                "pattern": "\\QMETA-INF/services/org.testcontainers.dockerclient.DockerClientProviderStrategy\\E",
+                "condition": {
+                    "typeReachable": "org.testcontainers.DockerClientFactory"
+                }
+            },
+            {
+                "pattern": "\\Qcom/github/dockerjava/zerodep/shaded/org/apache/hc/client5/version.properties\\E",
+                "condition": {
+                    "typeReachable": "com.github.dockerjava.zerodep.shaded.org.apache.hc.core5.util.VersionInfo"
+                }
+            },
+            {
+                "pattern": "\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E",
```

Comment:
Why is this class added as a resource? Is it necessary?

Reply:
I just used the output of the agent as-is.

I tried without the new entries and it works just as well, thanks for the catch!